corehq/apps/es/case_search.py

```diff
@@ -31,7 +31,7 @@
 
 
 class CaseSearchES(CaseES):
-    index = CASE_SEARCH_ALIAS
+    index = "case_search"
```

Comment:
It could be. This should be one of the keys in `ES_META`, it's neither an alias, nor and index name. Since `CASE_SEARCH_ALIAS` may no longer evaluate to this value, I had to change it. It's consistent with all other `HQESQuery` classes, so I let this be.